<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSCDeviceList</class>
 <widget class="QWidget" name="VSCDeviceList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>584</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbSite">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/action/resources/site.png</normaloff>:/action/resources/site.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Arial Unicode MS</family>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>Site</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRecorder">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">selection-background-color: rgb(0, 255, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/device/resources/computer.png</normaloff>:/device/resources/computer.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>Recorder</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCamera">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Add IP Camera</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/device/resources/camera.png</normaloff>:/device/resources/camera.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>Camera</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSurveillance">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Add View</string>
         </property>
         <property name="statusTip">
          <string>2</string>
         </property>
         <property name="whatsThis">
          <string>3</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/action/resources/surveillance.png</normaloff>:/action/resources/surveillance.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>View</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSearch">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Search IP Camera</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/action/resources/search.png</normaloff>:/action/resources/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbPlayback">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Playback </string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/action/resources/videosearch.png</normaloff>:/action/resources/videosearch.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>Playback </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VSCDeviceTrash" name="pbTrash">
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>40</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Drop IP Camera to Here</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="vscmainwindows.qrc">
           <normaloff>:/action/resources/user-trash-2.png</normaloff>:/action/resources/user-trash-2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;Arial Unicode MS&quot;;</string>
         </property>
         <property name="text">
          <string>Trush</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="VSCDeviceTree" name="treeWidget">
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerItem</enum>
       </property>
       <property name="indentation">
        <number>30</number>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>VS Cloud</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Sites</string>
        </property>
        <property name="icon">
         <iconset resource="vscmainwindows.qrc">
          <normaloff>:/action/resources/site.png</normaloff>:/action/resources/site.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Recorders</string>
        </property>
        <property name="icon">
         <iconset resource="vscmainwindows.qrc">
          <normaloff>:/device/resources/computer.png</normaloff>:/device/resources/computer.png</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cameras</string>
        </property>
        <property name="icon">
         <iconset resource="vscmainwindows.qrc">
          <normaloff>:/device/resources/camera.png</normaloff>:/device/resources/camera.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Views</string>
        </property>
        <property name="icon">
         <iconset resource="vscmainwindows.qrc">
          <normaloff>:/device/resources/view.png</normaloff>:/device/resources/view.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Disks</string>
        </property>
        <property name="icon">
         <iconset resource="vscmainwindows.qrc">
          <normaloff>:/device/resources/harddisk.png</normaloff>:/device/resources/harddisk.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Virtual Cameras</string>
        </property>
        <property name="icon">
         <iconset resource="vscmainwindows.qrc">
          <normaloff>:/device/resources/virtualipc.png</normaloff>:/device/resources/virtualipc.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>VSCDeviceTree</class>
   <extends>QTreeWidget</extends>
   <header>vscdevicetree.h</header>
  </customwidget>
  <customwidget>
   <class>VSCDeviceTrash</class>
   <extends>QPushButton</extends>
   <header>vscdevicetrash.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vscmainwindows.qrc"/>
 </resources>
 <connections/>
</ui>
